cli/volume.go

```diff
@@ -223,8 +223,8 @@ type VolumeInspectCommand struct {
 func (v *VolumeInspectCommand) Init(c *Cli) {
 	v.cli = c
 	v.cmd = &cobra.Command{
-		Use:   "inspect [OPTIONS] NAME",
-		Short: "Inspect a pouch volume",
+		Use:   "inspect [OPTIONS] Volume [Volume...]",
+		Short: "Inspect pouch one or more volumes",
```

Comment:
I guess "Inspect one or more pouch volumes" is better.

Reply:
thanks a lot for your work, i think we should also fix `network inspect` command !